spec/requests/users_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  before do
+    stub_request(:any, /localhost:9250/)
+  end
+  
+  let!(:user_a) { FactoryGirl.create(:user) }
+  let!(:user_b) { FactoryGirl.create(:user) }
+  
+  describe 'index' do
+    context 'when not signed in' do
+      it 'redirects to sign in' do
+        get '/users'
+        expect(response).to redirect_to("/users/sign_in")
+      end
+    end
+    
+    context 'when signed in' do
+      before do
+        login_as user_b
+      end
+      
+      it 'displays list of users' do
+        get '/users'
```

Comment:
These tests seem like they belong in the controller specs...

Reply:
i agree, but looking at request tests on relishapp.com it shows these examples so i just included them again.

Reply:
hmm, maybe that's the new thing and we don't need controller tests anymore...